### PR TITLE
Change some phrases for brain damage and add some more

### PR DESCRIPTION
**Pull request by Anasari**
Removed / changed phrases that broke the rules. Including IC in OOC (Griffin me!!), Netspeak (lol2cat), and emoticon (;-;). 

Add in some original SS13 memes to compensate. 

Feedback welcome

🆑Ansari
tweak: Change some brain damage phrases. Add a lot of new one.
/🆑

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
I'm inclined to say no to this; they've always been like this; brain damage lines are kinda allowed to break immersion because the person in question is brain-damaged; they're allowed/going to stay stupid/non-sensical things.

**Comment by Anasari** [comment]
I think that a bit of immersion breaking would be nice. But I think that presence of rule-breaking phrase within the brain damage speech would set a bad example. i.e. "why am i not allowed to netspeak when the code let me do so."

A more conservative PR that only remove the emoticon might work.

**Comment by taukausanake** [comment]
Yeah, I'd remove the emoticon personally. And the 4Chan arrow as in the PR. Everything else I'd keep though. I read netspeak as it appears as it sounds. I don't think there's anything wrong with that if someone is literally retarded.
No idea why you'd remove fus ro dah. Stupid yes but it's not breaking any rules.

Also I'd say to add those new lines too. Sound great.

**Comment by Spacemanspark** [comment]
:+1: The netspeak and whatnot has been grinding my gears for years, honestly.
@Fox-McCloud I'd argue that just because something's been that way for years, doesn't mean it's a valuable addition to the code/ server. We have server rules that specifically block language such as smileys or netspeak, but having this contradicts that and confuses people.

**Comment by Anasari** [comment]
I re-added fus ro dah as it doesn't actually break rules, as silly as it is. 

Added in a bunch of original contents that refer SS13. if any of them are problematic please tell me.

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -102,13 +102,18 @@
...
-								 "VOXES caN't LOVE")
-
-			var/list/s2 = list("ONI'SOMA",
-							   "fookin tangerines!!!",
+								 "VOXES caN't LOVE",
+								 "my dad own this station",
+								 "the CHef put CHLORINE in our food!",
+								 "the monkey have TASER ARMS!",
+								 "cargo ate all my cookie points",
+								 "EI'NATH!")
```
Comment (taukausanake) [inline code/modules/mob/living/carbon/human/life.dm]: Hmmmmmmm, maybe not this. Unless you can misspell it?

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -292,7 +301,7 @@
 		var/skip_contents_check = 0 //rigsuit snowflake, oxygen tanks aren't stored inside the mob, so the 'contents.Find' check has to be skipped.
 
 		if(!get_organ_slot("breathing_tube"))
-			if(!(wear_mask && wear_mask.flags & AIRTIGHT)) //if NOT (wear_mask AND wear_mask.flags CONTAIN AIRTIGHT)
+			if(!(wear_mask && wear_mask.flags & AIRTIfGHT)) //if NOT (wear_mask AND wear_mask.flags CONTAIN AIRTIGHT)
```
Comment (taukausanake) [inline code/modules/mob/living/carbon/human/life.dm]: Uhhhhhhh...

Comment (taukausanake) [reply code/modules/mob/living/carbon/human/life.dm]: Oh wow fixed it before I posted it

**Comment by Anasari** [comment]
I'm honestly not too certain on whether "griffin" or "griefing" shall be removed. After all, we do have someone called the Griffon.

The emoticons / netspeak however, I think it have to be removed so we are in fact consistent with our own rules

**Comment by uraniummeltdown** [comment]
The OOC in IC phrases are classics, don't remove them.
Also why change "fucking 4rries" to "fucking tangerines"? Make it furries
Furry is an actual word and not some kind of slur

**Comment by alexkar598** [comment]
The one you can't say in real life shoul probably go but the ones that are IOC in ic are fine since your character is mentally damaged leading to it thinking it is in a game or saying things that dose not make sense

**Comment by FalseIncarnate** [comment]
I think https://github.com/ParadiseSS13/Paradise/pull/7005 set a bit of a precedent regarding in-game message contents. We removed a word that would get someone banned if they used it on the server, so I can see that same argument applying for removing the emoticon and MAYBE the netspeek, though "Griffin me" is not really worthy of being dropped.

I'd recommend changing the "WHERE MY OMNIZINE" to "WHERE MY TRICORD", and maybe change the "pi hole" option to something someone might not mistake for pee hole at a quick glance. I know its not, but better safe than sorry on that.

**Comment by Doukhan** [comment]
I like the new phrases but the old ones are indeed classics, that they're only said by the brain-damaged is half the joke as well so I don't see why we can't just let it slide.
@FalseIncarnate I get where you're coming from but SS13 is pretty far from PG13, what with the bloody dismemberment and all. I like to pretend that most people who play it are mature enough to appreciate the humour. No need for sanitizing it IMO.

**Comment by Anasari** [comment]
The fact we have blatantly rule-breaking phrase uttered by the brain damaged lead to reduced credibility when we do try to enforce our rules. I've had more than one time when someone was pinged for emoticon / netspeech and they cited our own code against us.

While that won't excuse them from breaking the rules, they do have a point. It would be hypocritical to enforce a rule on people while breaking it with our own code, so to speak.  In fact, there's an exact precedent for this: https://github.com/ParadiseSS13/Paradise/pull/6525 

PR 6525 removed phrase that were blatantly OOC in IC and referred OOC concepts. I don't see why brain damage message couldn't undergo the same changes so we are consistent with our own rules and desired RP standard, either.

I'll make the omnizine phrase references (non-existent on para) baymed and change pi hole to pie hole. As much as I'd like to make a 3.14 joke, False has a good point that it might be mistaken for something way more explict.

**Comment by tigercat2000** [comment]
`change the "pi hole" option to something someone might not mistake for pee hole at a quick glance. `
`change pi hole to pie hole`
That would be be blatantly making it worse...

**Comment by Anasari** [comment]
that's it, no hole.

**Comment by Tayyyyyyy** [comment]
Since nobody has commented on this yet, I really like the new phrases. I think they're much more creative and fit the atmosphere of the server better.

**Comment by taukausanake** [comment]
@Tayyyyyyy 
> nobody has commented on this yet

**Me**
> Also I'd say to add those new lines too. Sound great.

👎 👎 👎 👎 👎

File: code/modules/mob/living/carbon/human/life.dm [hunk 3]
```diff
@@ -94,32 +94,49 @@
 	if(getBrainLoss() >= 60 && stat != DEAD)
 		speech_problem_flag = 1
 		if(prob(3))
-			var/list/s1 = list("IM A PONY NEEEEEEIIIIIIIIIGH",
+			var/list/s1 = list("IM A [pick("PONY","LIZARD","taJaran","kitty","Vulpakin","drASK","BIRDIE","voxxie","race car","combat meCH")] [pick("NEEEEEEIIIIIIIIIGH","sKREEEEEE","MEOW","NYA~","rawr","Barkbark","Hissssss","vROOOOOM","pewpew","choo Choo")]!",
```
Comment (FalseIncarnate) [inline code/modules/mob/living/carbon/human/life.dm]: "IM A LIZARD choo Choo!"
"IM A combat meCH NYA~!"
"IM A drASK pewpew!"

These utterly random combinations really make this part of the feature realistic.

Comment (taukausanake) [reply code/modules/mob/living/carbon/human/life.dm]: Add spaceship (or spessship) to the first part please

**Comment by adamkad1** [comment]
netspeak allways bothered me. will you ban someone from real life for netspeaking outside of games? people who do that might not have too good brains but still some people do that

**Comment by taukausanake** [comment]
I just say "lol" in real life. It pisses my sister off but no one else seems to care. Like it isn't an issue so long as it doesn't sound retarded to the ears. Like spelling it out; just say it as one word

**Comment by tigercat2000** [comment]
👍

**Pull request merged**